client/scss/critical.scss

```diff
@@ -37,6 +37,7 @@
 @import 'components/exhibition_promo';
 @import 'components/exhibition_hero';
 @import 'components/image_viewer';
+@import 'components/gif_video';
```

Comment:
Arguable - why critical?

Reply:
Point four above –  prevents flash of massive poster frame

Reply:
Hmm. 

Might need to chat about what critical is for - we aren't going to have these on 95% of pages, so we're serving up a bit of CSS to everyone repeatedly for a small percentage of worth.

If the point is to avoid flash at all cost, then we've succeeded.

If the point it to make sure critical content is rendered sans-flash, and allowing non-critical content to have a tiny flash... I suppose the question to know if we've succeeded is, "Are vidiGifs critical content?".